Puzzlepart/prosjektportalen365
# Add tag to deliveries elements on timeline

### Your checklist for this pull request


### Description

Add ability to set and see tags for deliveries elements on timeline similar as to normal timeline elements.

| Before | After |
| ------- | ----- |
| <img width="297" alt="image" src="https://user-images.githubusercontent.com/28678468/209713714-151fd593-a487-4b1e-9bc1-e2440ea4c36c.png"> | <img width="291" alt="image" src="https://user-images.githubusercontent.com/28678468/209713783-95c1309d-3a83-4500-bb6e-7b6a7202eef3.png"> |

- Also fixed problem where an element without type caused timeline to not load.

### How to test

1. Add `GtTag` SiteField to a Projectdeliveries list in a Project
2. Add data to GtTag
3. Check that timeline element has tag in upper right corner (Project and Portfolio)

### Relevant issues (if applicable)

#917
